client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -1605,21 +1605,29 @@ class LifecycleManager(appId: String, val conf: CelebornConf) extends RpcEndpoin
     }
   }
 
-  private def requestCommitFiles(
+  private def requestCommitFilesWithRetry(
       endpoint: RpcEndpointRef,
       message: CommitFiles): CommitFilesResponse = {
-    try {
-      endpoint.askSync[CommitFilesResponse](message)
-    } catch {
-      case e: Exception =>
-        logError(s"AskSync CommitFiles for ${message.shuffleId} failed.", e)
-        CommitFilesResponse(
-          StatusCode.FAILED,
-          List.empty.asJava,
-          List.empty.asJava,
-          message.masterIds,
-          message.slaveIds)
+    val maxRetries = conf.requestCommitFilesMaxRetries
+    var retryTimes = 0
+    while (retryTimes < maxRetries) {
+      try {
+        return endpoint.askSync[CommitFilesResponse](message)
+      } catch {
+        case e: Exception =>
+          retryTimes += 1
```

Comment:
1. whether should need judge exception type like timeoutException? 
2. whether  should wait somestimes before next retry?

Reply:
1. I think we should retry regardless the Exception type.
2. For immediate Exceptions it's useful, but for TimeoutException it's unnecessary.